utils/searchHandler.ts

```diff
@@ -63,41 +49,14 @@ export const handleSearch = async (
       return null;
     }
 
-    return { search_id, imgURL };
+    return { search_id, imgURL, results };
   } catch (error) {
     console.error("Search failed:", error);
     setIsLoading(false);
     return null;
   }
 };
 
-export const addProductsToDatabase = async (
```

Comment:
@Anubhav-Goyal01 so now how are we adding the products to the DB?

Reply:
I changed the backend code, we have all the db connections there only.